DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: workflow.portfolio.parsing
 Title: Reads, cleans, and reexports portfolios for PACTA
-Version: 0.0.0.9000
+Version: 0.1.0
```

Comment:
I have a general question to @AlexAxthelm  and @cjyetman: what is our "release" process? 
* Initially, we would only bump `r2dii.*` version numbers once they were accepted to CRAN
* We then entered a situation where we "released" the `pacta.*` packages when they were used to build a "release candidate" for Transition Monitor. This was more or less just to facilitate "easily" re-building the image based on the same package versions, to be able to simulate, e.g. re-running a COP portfolio. 

As far as I can tell, this is now serving neither of the above, but instead the `WebApp` circumstance, so I guess this perhaps is a call to align on our definitions of a "release", otherwise I fear it ends up being a meaningless term. 

NIT: I will extract this into an issue in the `practices` repo.

Reply:
Definitely hear that. My goal with this change is to communicate:
* Not at v1.0 yet (0.*1*.0)
* This is shaped like a release (no dev tag / `.9000`)

Reply:
Indeed. Then I think we have a soup of different "release" strategies and should definitely align on that before we merge a new one into the mix.